sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md

```diff
@@ -1,20 +1,19 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
+This release is the first stable release of the Storage Mover Management library.
 
-### Bugs Fixed
-
-### Other Changes
+### Features Added
+-
+- Upgraded API version to 2021-01-01.
```

Comment:
Per the swagger PR https://github.com/Azure/azure-rest-api-specs/pull/21926, the GA API version should be "2023-03-01"

Reply:
This is a place holder for now, will replace it after we get the stable version. Thank you for point it out.